metron-platform/metron-enrichment/src/main/scripts/start_enrichment_topology.sh

```diff
@@ -20,7 +20,7 @@ METRON_VERSION=${project.version}
 METRON_HOME=/usr/metron/$METRON_VERSION
 TOPOLOGY_JAR=${project.artifactId}-$METRON_VERSION-uber.jar
 
-# there are two enrichment topologies.  by default, the split-join enrichment topology is executed
+# There are two enrichment topologies. By default, the unified enrichment topology is executed. Split-join is now deprecated.
 SPLIT_JOIN_ARGS="--remote $METRON_HOME/flux/enrichment/remote-splitjoin.yaml --filter $METRON_HOME/config/enrichment-splitjoin.properties"
 UNIFIED_ARGS="--remote $METRON_HOME/flux/enrichment/remote-unified.yaml --filter $METRON_HOME/config/enrichment-unified.properties"
```

Comment:
We also need to change line 28 of this file so that it defaults to Unified.
```
ARGS=${@:-$UNIFIED_ARGS}
```

Reply:
> @mmiklavc Should we add a small blurb in `Upgrading.md` to document how users who are upgrading can revert to the existing functionality?

Hrm, our upgrading doc appears to list items on a per-release basis, and we don't have anything new since 0.5.0. Here's what I'll do. Add an upgrading from 0.6.0 to 0.6.1 notice and make we change the version accordingly in the next release.